java/ql/test/query-tests/security/CWE-074/JndiInjectionTest.ql

```diff
@@ -0,0 +1,22 @@
+import java
+import semmle.code.java.dataflow.TaintTracking
+import semmle.code.java.dataflow.FlowSources
```

Comment:
Superfluous, right?
```suggestion
```

Reply:
Indeed, those should have been removed when `JndiInjectionFlowConfig` was reused here. Good catch.